pallets/xvm/src/benchmarking.rs

```diff
@@ -24,7 +24,11 @@ use parity_scale_codec::Encode;
 use sp_core::H160;
 use sp_runtime::MultiAddress;
 
-#[benchmarks]
+use astar_primitives::Balance;
+
+#[benchmarks(
+    where <T as pallet_contracts::Config>::Currency: Currency<T::AccountId, Balance = Balance>,
+)]
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

LGTM!



<!-- This is an auto-generated comment by OSS CodeRabbit -->